sqlmesh/core/engine_adapter/mysql.py

```diff
@@ -73,3 +80,52 @@ def _get_data_objects(self, schema_name: SchemaName) -> t.List[DataObject]:
             )
             for row in df.itertuples()
         ]
+
+    def _create_comments(
+        self,
+        table_name: TableName,
+        table_comment: t.Optional[str] = None,
+        column_comments: t.Optional[t.Dict[str, str]] = None,
+        table_kind: str = "TABLE",
+    ) -> None:
+        """
+        Executes commands to create table and column comments.
+        """
+        table = exp.to_table(table_name)
+        table_sql = table.sql(dialect=self.dialect, identify=True)
+
+        if table_comment:
+            try:
+                self.execute(
+                    f"ALTER TABLE {table_sql} COMMENT = '{table_comment}'",
+                )
+            except:
```

Comment:
I think generally it's not a good idea to catch all possible exceptions. Can we at least do `except Exception`?

Reply:
This try/except is motivated by permissions errors, which should all come from the database connector library (and not be raised as SQLMesh exceptions by sqlmesh itself). 

What do you think about raising for any sqlmesh exception and warning for everything else?

```
            try:
                ...
            except SQLMeshError as e:
                raise e
            except Exception as e:
                logger.warning(...)
```